Command/DropDatabaseDoctrineCommand.php

```
@@ -92,6 +92,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
         // Reopen connection without database name set
         // as some vendors do not allow dropping the database connected to.
         $connection->close();
+
+        // If the database config comes from a URL,
+        // ensure that the database name is not set.
+        if (isset($params['url'])) {
+            $params['url'] = str_replace('/'.basename($params['url']), '', $params['url']);
```

Should probably throw an exception instead